src/noisepy/seis/datatypes.py

```diff
@@ -168,5 +168,5 @@ class NoiseFFT:
     fft: np.ndarray
     std: np.ndarray
     fft_time: np.ndarray
-    SegmentCount: int
-    Length: int
+    segment_count: int
```

Comment:
could we call that "window_count" i think seismologists prefer the use of "window" rather than "segment"